internal/uidriver/glfw/hideconsole_windows.go

```diff
@@ -60,6 +61,14 @@ func getConsoleWindow() (uintptr, error) {
 	return r, nil
 }
 
+func freeConsole() error {
+	_, _, e := freeConsoleWindowProc.Call()
+	if e != nil && e.(windows.Errno) != 0 {
+		return fmt.Errorf("ui: FreeConsole failed: %d", e)
+	}
+	return nil
+}
+
 func showWindowAsync(hwnd uintptr, show int) error {
```

Comment:
Do we still use this?

Reply:
No, as mentioned !
Should I remove it ?

Reply:
Yes, if this is not used anywhere.

Reply:
Done!